vendor/bass/basswma.h

```diff
@@ -55,8 +55,6 @@ typedef DWORD HWMENCODE;	// WMA encoding handle
 #define BASS_WMA_ENCODE_SCRIPT		0x20000	// set script (mid-stream tags) in the WMA encoding
 #define BASS_WMA_ENCODE_QUEUE		0x40000	// queue data to feed encoder asynchronously
 #define BASS_WMA_ENCODE_SOURCE		0x80000	// use a BASS channel as source
-#define BASS_WMA_ENCODE_VOICE		0x100000 // WMA Voice

```

Comment:
He just needs to merge master branch, @qaisjp

Reply:
the change won't be shown here unless it's a manual one.

Reply:
Yes, unintended. However, I think that Dutchman is right because this appeared right after this commit: https://github.com/multitheftauto/mtasa-blue/commit/bf99a8303e28fc54b4d5c0fa8dc4e1f8b8a2c39e

Reply:
Yes, unintended. However, I think that Dutchman is right because this appeared right after this commit: https://github.com/multitheftauto/mtasa-blue/commit/bf99a8303e28fc54b4d5c0fa8dc4e1f8b8a2c39e

Reply:
git doesn't randomly introduce changes from other branches. It was introduced in this commit d43cd0953c35220e2886e8d03d2df26afcebd574

Reply:
Done